docs/source/examples/learning-depolarizing-noise.md

```diff
@@ -50,7 +50,7 @@ Here we use a simple Rx-Rz-CNOT circuit, with an (optional) seed for reproducibi
 
 ```{code-cell} ipython3
 circuit = random_x_z_cnot_circuit(
-    LineQubit.range(2), n_moments=10, random_state=np.random.RandomState(1)
+    LineQubit.range(2), n_moments=5, random_state=np.random.RandomState(1)
```

Comment:
Why is it that this change does not require a change to the data that is loaded in further into the file? Naively, I would have thought if we are changing the circuit, then the PEC data would need to change as well. Does the PEC data consist of representations of the gates, which does not change here?

Reply:
The change is to match the circuits used to generate the file. The file was generated separately from this notebook and is already used in `test_learning.py` to make the tests run efficiently.

Reply:
Ahh okay that makes sense! Thanks for the clarification. Can you comment on what's in the data file? (just for the sake of me learning!)